stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,798 @@
+     <span class="fd-list-attribute">Third Attribute Very Very Very Very Very Very long</span>
+   </div>
+   </div>
+   </div>
+   </div>
+    </a> 
+  </li>
+  <li role="listitem" tabindex="-1" class="fd-list__item fd-list__item--link is-selected">
+    <a tabindex="0" class="fd-list__link fd-list__link--navigation-indicator is-navigated">
+    <div class="fd-list__content">
+    <div class="fd-list-optional">
+    <span class="fd-list-optional__text">Optional inline text</span>
+    </div>
+    <div class="fd-list__object fd-list__object--top-row">
+    <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--thumbnail" style="background-image: url('http://lorempixel.com/400/400/nature/4/')" role="presentation" aria-label="John Doe"></span>
+    <div class="fd-list__object fd-list__object--top-left fd-object-identifier">
+      <p class="fd-object-identifier__title">
+    Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+     </p>
+     </div>
+    <div class="fd-list__object fd-list__object--top-right fd-object-number fd-object-number--positive">
+    <span class="fd-object-number__text">956.00</span>
+    <span class="fd-object-number__unit">Euro</span>
+    </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom">
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-right">
+      <span class="fd-object-status fd-object-status--positive">First status</span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Second Attribute</span>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-right">
+   <span class="fd-object-status fd-object-status--positive">Second status</span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Third Attribute</span>
+   </div>
+   </div>
+   </div>
+   </div>
+    </a> 
+  </li>
+</ul>
+</div>
+`;
+
+navigationIndicator.storyName = 'Navigation Indicator';
+
+navigationIndicator.parameters = {
+    docs: {
+        iframeHeight: 355,
+        storyDescription: `
+For a object list with navigation links and navigation indicator add \`fd-list--navigation\` \`fd-list--navigation-indication\` along with \`fd-list\` and \`fd-list--object\` in list and \`fd-list__link\`, \`fd-list__link--navigation-indicator\`, \`fd-list--navigation\` modifier class to the link(anchor) and the
+\`fd-list__item--link\` modifier class to the list element. that contain links.This will help all items to be navigable with navigation indicator.
+`
+    }
+};
+
+export const selection = () => `
+<h4>Object List Item With Row Selection</h4>
+<div style="max-width: 450px">
+<ul class="fd-list fd-list--object fd-list--selection fd-list__row-selection" role="list">
```

Comment:
Selection lists should have listbox role:
```suggestion
<ul class="fd-list fd-list--object fd-list--selection fd-list__row-selection" role="listbox">
```

Reply:
updated